Helper/Utils.cs

```
@@ -0,0 +1,30 @@
+﻿using System.Text;
+
+namespace THESSA.Helper;
+
+public static class Utils
+{
+    public static string DecodeContentBase64(string content)
+    {
+        if (string.IsNullOrWhiteSpace(content)) 
+            throw new ArgumentNullException("Content cannot be null or Empty", nameof(content));
+
+        // Normalize by Removing escape characters and trimming whitespaces
+
+        var normalize = content
+            .Replace("\n", "")
+            .Replace("\r", "")
+            .Replace("\\n", "")
+            .Trim();
+
+        try
+        {
+            byte[] data = Convert.FromBase64String(normalize);
+            return Encoding.UTF8.GetString(data);
+        }
+        catch (Exception ex)
+        {
```

26: Catching general Exception is not recommended; it may mask unexpected errors unrelated to Base64 decoding.